src/client/components/App.jsx

```diff
@@ -34,6 +35,6 @@ const App = withContextProviders(() => {
             </Switch>
         </BrowserRouter>
     );
-}, [InnrapporteringProvider, PersonProvider, AuthProvider, EasterEggProvider]);
+}, [InnrapporteringProvider, PersonerProvider, PersonProvider, AuthProvider, EasterEggProvider]);
```

Comment:
`PersonerProvider`/`PersonerContext` og `PersonProvider`/`PersonContext` har kanskje litt for like navn? Hva med `PersonoversiktContext`/`PersonoversiktProvider`?

Reply:
Eller `OversiktContext` som du nevnte over :)

Reply:
Jeg liker `PersonoversiktContext` :+1: